src/locales/en.ts

```diff
@@ -135,6 +135,7 @@ export default {
       rak: 'RAK\nHotspot Miner',
       nebrain: 'Nebra\nIndoor Hotspot',
       nebraout: 'Nebra\nOutdoor Hotspot',
+      syncrobit: 'SyncroB.it Hotspot',
```

Comment:
You could also change this to `SyncroB.it\nHotspot` to make sure it fits:

![Screen Shot 2021-02-19 at 9 59 19 PM](https://user-images.githubusercontent.com/6345962/108585533-d02f8680-72fd-11eb-9f10-521d5b4b7f9d.png)